tests/vasp/flows/test_core.py

```diff
@@ -15,8 +17,8 @@ def test_double_relax(mock_vasp, clean_dir, si_structure):
 
     # settings passed to fake_run_vasp; adjust these to check for certain INCAR settings
     fake_run_vasp_kwargs = {
-        "relax 1": {"incar_settings": ["NSW", "ISMEAR"]},
-        "relax 2": {"incar_settings": ["NSW", "ISMEAR"]},
+        "relax 1": {"incar_settings": ["NSW", "ISMEAR", "LREAL"]},
```

Comment:
The tests are failing because of this line. You are validating on `LREAL` which will compare the written files vs reference files.

The reference files contain `LREAL = Auto` but some of your calculations contain `LREAL = False` so they don't match.

Better would be to remove the check here, and manually check the task document as you are doing below.

Reply:
Thanks for pointing that out. Now, `LREAL` is getting set to `True` somewhere, which is... peculiar... seeing that the default is 'Auto'. I'm digging into that now.

Reply:
Can I check the task doc if the mock vasp run is copying over the INCAR? Wouldn't the `orig_inputs` just be pulled from that INCAR then? Similarly, `.input.parameters` is from the `vasprun.xml` which of course is not from the modified VASP run. 

I'm trying to find out where I can check for the new input settings.

Reply:
Ah sorry this is getting so complicated. Yeah, I see what you mean. There is no easy way to validate the written inputs without creating a new directory of reference test data.

I guess the best thing to do is to create a new folder of test data and symlink the files that are the same between the two folders (to save space), and just include a new INCAR file with your changes.

Sorry for the hassle!

Reply:
Well, I just feel validated that I understood what was going on! I'll take it!

That's more or less what I figured. I'll give that a go in the morning over here in the states.